### PR TITLE
Created Profile UI and Setup nested routes

### PR DESCRIPTION
**Pull request by gbudjeakp**
fdestination
notification
account

### CODE REVIEW

File: client/src/pages/FavouriteDestinations/FavoriteDestinantions.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <Grid className={classes.root}>
+          <Button variant="outlined" style={{float: "right", marginRight: 25}}>Explore</Button>
+          <Typography className={classes.title} variant="h5">Favorite Destinations</Typography>
+            <Grid
+   container
+   spacing={3}
+   justify="center"
+   style={{  marginTop: 10 }} >
+   {places.map((place, index) => (
+     <Grid item key={index}>
```
Comment (johnsciutto) [inline client/src/pages/FavouriteDestinations/FavoriteDestinantions.js]: It's usually not a good idea to use the `index` as keys to map over components in React. You could run into [some issues](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318). If the `place` objects don't have unique ids, use something that will always be unique about them.

**Review by johnsciutto — COMMENTED** [review]
As a general rule, try to limit the use of inline styles if you're using classes and `useStyles` from Material UI. It makes the CSS harder to debug/change if there are different sources of truth with different "specificity" values.

File: client/src/pages/FavouriteDestinations/FavoriteDestinantions.js (hunk 1 above)
Comment (gbudjeakp) [reply client/src/pages/FavouriteDestinations/FavoriteDestinantions.js]: I'll make a push effecting the changes immediately.

File: client/src/pages/FavoriteDestinantions.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        <CustomSmallerCheckBox
+          checked={checked}
+          onChange={(e) => setChecked(e.target.checked)}
+          icon={<Favorite style={{ color: "white" }} />}
+          checkedIcon={<Favorite style={{ color: "#FFA000" }} />}
+          classes={{ root: classes.customCheckBoxRoot }}
+        />
+      </div>
+    );
+  }
```
Comment (sina-jamshidi) [inline client/src/pages/FavoriteDestinantions.js]: Seems like we can abstract this out to it's own component in `client/src/components/`. Also the `classes` are not used so you can remove that part

**Review by sina-jamshidi — APPROVED** [review]
Looks good once comments are addressed!

**Pull request merged**